components/__design__/list.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { CSSProperties } from 'react';
+
+const listContainer = {
+  padding: '1.5rem',
+} as CSSProperties;
```

Comment:
```suggestion
} satisfies CSSProperties;
```

Technically speaking this object is **not** `CSSProperties` but it satisfies it.

Reply:
Perfect and updated!